compiler/src/dotty/tools/dotc/typer/Applications.scala

```diff
@@ -92,7 +93,7 @@ object Applications {
     def getTp = extractorMemberType(unapplyResult, nme.get, pos)
 
     def fail = {
-      ctx.error(i"$unapplyResult is not a valid result type of an $unapplyName method of an extractor", pos)
+      ctx.error(InvalidUnapplyReturnType(unapplyResult, unapplyName))
```

Comment:
How is the `pos` being passed here? (sorry for the probably naive question, I am just starting on those error messages)

Reply:
I actually misunderstood this value's usage, fixed this.